tests/features/javascript/authentication.js

```diff
@@ -0,0 +1,25 @@
+const { Given, Then, When } = require('cucumber')
+
+Given('an anonymous user', function () {
+  // Write code here that turns the phrase above into concrete actions
+  return 'pending'
+})
+
+Given('an existing user', function () {
+  // Write code here that turns the phrase above into concrete actions
+  return 'pending'
+})
+
+When('I sign up with a valid email and password', function () {
+  // Write code here that turns the phrase above into concrete actions
+  return 'pending'
+})
+Then('I should be logged in', function () {
+  // Write code here that turns the phrase above into concrete actions
+  return 'pending'
+})
+
+Then('I should be able to log out', function () {
+  // Write code here that turns the phrase above into concrete actions
+  return 'pending'
+})
```

Comment:
These definitions match (essentially regex) against any feature line. Perhaps we would do a one-to-one `.feature` to `step definition` - not sure about the code organization tbh